client/my-sites/upgrades/checkout/privacy-protection.jsx

```diff
@@ -73,8 +76,35 @@ module.exports = React.createClass( {
 			priceForPrivacyText = this.translate( 'Privacy protection keeps your information hidden when these domains are searched for in the public database. You can use it for no extra cost.' );
 		}
 
+		if ( abtest( 'privacyCheckbox' ) === 'checkbox' ) {
+			return (
+				<div>
+					<Card className="checkout__privacy-protection-checkbox">
+						<input type="checkbox" onChange={ this.props.onCheckboxChange } checked={ this.props.isChecked } />
+						<div className="privacy-protection-checkbox__description">
+							<strong className="checkout__privacy-protection-checkbox-heading">{ this.translate( 'Please keep my information private.', { textOnly: true } ) }</strong>
+							<p className="checkout__privacy-protection-checkbox-text">{ priceForPrivacyText }</p>
+							<a href="" onClick={ this.handleDialogOpen }>Learn more about Privacy Protection.</a>
+						</div>
+						<div>
+							<Gridicon icon="lock" size={ 48 } />
+						</div>
+					</Card>
+					<PrivacyProtectionDialog
```

Comment:
This means that `PrivacyProtectionDialog` is duplicated, but since this is just a test it's probably ok.